src/porepy/fracs/structured.py

```diff
@@ -1,7 +1,11 @@
-"""
-Module for creating fractured cartesian grids in 2- and 3-dimensions.
+"""Module for creating fractured cartesian grids in 2 and 3 dimensions.
 
 The functions in this module can be accessed through the meshing wrapper module.
+
+Todo:
+    Since this module contains only private methods, consider making the whole
```

Comment:
Agree - and that applies to many other modules in this directory. This is not for this PR, though.